### ISSUE TITLE
Lecture 01

### ISSUE THREAD

**Issue by rdt2992** [rdt2992/ML#1]
Machine Learning : explicit programming이 한계를 보임(많은 rule을 가진 경우 개발자가 일일이 프로글래밍하기 어려움) => data들을 통해 컴퓨터가 배우는 것을 고안

1.supervised learning : label이 정해진 데이터를 학습(답을 알려주는 data로 학습, training data set) 일반적인 문제들(사진, 스팸메일, 점수예측 등) Training set을 이용해 모르는 것을 알아 냄 1)regression : 일정 범위 내에 답이 존재 ex)점수 2)binary classification : 2가지로 분류 ex)pass/fail 3)multi-label classification : 여러가지 label로 분류 ex)A B C D E

2.unsupervised learning : un-labeled data, 유사한 것들끼리 묶음 ex)word clustering
